### DISCUSSION TITLE
Readd Profile at Hook login.auth

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#9659] [Feature Requests]
In the old directus verions you received the profile in the auth.login this isn't anymore.
I found a "hacky" solution, but i think this is not how it is supposed to be:

https://github.com/directus/directus/discussions/9651

**Comment by NilsBaumgartner1994** [comment]
With an gist
https://gist.github.com/NilsBaumgartner1994/be374e0aef5d92012451e84d36677dac

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
